openfl/federated/task/runner_keras.py

```diff
@@ -98,7 +98,8 @@ def train(
             raise KeyError("metrics must be defined")
 
         # rebuild model with updated weights
-        self.rebuild_model(round_num, input_tensor_dict)
+        if round_num > 0:
```

Comment:
I believe this will result in the aggregator's initial model not being used for the first round. This will be problematic if a pretrained model is used as the basis for federated fine-tuning.